Tests/Functional/AllowedMethodsTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Tests\Functional;
+
+/**
+ * @author Ener-Getick <egetick@gmail.com>
+ */
+class AllowedMethodsTest extends WebTestCase
+{
+    public function setUp()
+    {
+        $this->client = $this->createClient(array('test_case' => 'ParamFetcher'));
```

Comment:
Looks a bit strange to search for the configuration in the `ParamFetcher` test. Let's use a dedicated section instead (makes it easier to find the right test case).